hack/hive-generate-config.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # This is the commit sha that the image was built from and ensures we use the correct configs for the release
-HIVE_IMAGE_COMMIT_HASH="56adaaa"
+HIVE_IMAGE_COMMIT_HASH="fec14dc"
```

Comment:
I see it is refering the right commit https://github.com/openshift/hive/commit/fec14dcf0cfc433c6bf141d9b25a66c6c652af37